<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnGenerateShadows">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Shadows</string>
    </property>
   </widget>
   <widget class="QLabel" name="picSelected">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>531</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../podanie_skan.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="picDecoded">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>541</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/Nosfreat/Desktop/voucher kurs fotografii 2024 Paweł Sieciński.jpg</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="comboEncodingType">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>320</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Type1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Type2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Type3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Type4</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtNumberOfShadows">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>350</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>No of shadows</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>300</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Encoding Type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectImage">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Image</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>1231</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ShadowsListGroup">
     <item>
      <widget class="QListWidget" name="listGeneratedSh">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ShadowsMoveGroup">
       <item>
        <widget class="QPushButton" name="btnSelectShadows">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMoveBack">
         <property name="text">
          <string>Move back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listSelectedSh">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="txtShadowThreshold">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>380</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Threshold</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDecode">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>440</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Decode</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1293</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
